From: Thomas Haller <thaller@redhat.com>
Date: Sun, 24 Jan 2016 16:40:29 +0100
Subject: libnm-gtk/tests: fix test failure in test-mobile-providers.c for non
 C locale

Test failed with different locale:

    LANG=de_DE.UTF-8 make check

https://bugzilla.gnome.org/show_bug.cgi?id=760966

(cherry-picked from commit d78403cba72bad8d76c5f4dfb5ee9d98ae04a561)
---
 po/POTFILES.in                              | 1 +
 src/libnm-gtk/tests/test-mobile-providers.c | 6 +++++-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 1af8d65..7378c4b 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -80,6 +80,7 @@ src/libnm-gtk/nm-mobile-wizard.c
 src/libnm-gtk/nm-ui-utils.c
 src/libnm-gtk/nm-vpn-password-dialog.c
 src/libnm-gtk/nm-wifi-dialog.c
+src/libnm-gtk/tests/test-mobile-providers.c
 [type: gettext/glade]src/libnm-gtk/wifi.ui
 src/main.c
 src/mb-menu-item.c
diff --git a/src/libnm-gtk/tests/test-mobile-providers.c b/src/libnm-gtk/tests/test-mobile-providers.c
index ea8b93c..310da14 100644
--- a/src/libnm-gtk/tests/test-mobile-providers.c
+++ b/src/libnm-gtk/tests/test-mobile-providers.c
@@ -13,9 +13,13 @@
  * Copyright (C) 2012 Aleksander Morgado <aleksander@gnu.org>
  */
 
+#include "config.h"
+
 #include <locale.h>
 #include <string.h>
 
+#include <glib/gi18n-lib.h>
+
 #include "nm-mobile-providers.h"
 
 #if defined TEST_DATA_DIR
@@ -317,7 +321,7 @@ ensure_movistar (NMAMobileProvider *provider)
 	}
 
 	/* Check access method name, APN and type */
-	g_assert_cmpstr (nma_mobile_access_method_get_name (method), ==, "Default");
+	g_assert_cmpstr (nma_mobile_access_method_get_name (method), ==, _("Default"));
 	g_assert_cmpint (nma_mobile_access_method_get_family (method), ==, NMA_MOBILE_FAMILY_3GPP);
 	g_assert_cmpstr (nma_mobile_access_method_get_3gpp_apn (method), ==, "movistar.es");
 
